mamedev/mame
# a2bus: Add support for Wico Trackball, Wico Command Control Joystick Adapter, and 4 paddles connected to the Sirius JoyPort to the Apple ][/][+

Adds support for these Apple ][ / ][+ controls:
- Wico Trackball
- Wico Command Control Joystick Adapter
- 4 paddles connected to the Sirius JoyPort

To test the Wico Command Control Adapter with 2 Atari-style joysticks:
mame apple2p -gameio wicojoy 1on1

Play Mode: Two Player
Play Ball!: New Game
Ensure that both players can play using the 1st and 2nd joysticks


To test the Wico Trackball control:
mame apple2p -sl2 wicotrackball apple2_flop_misc:spaceark
Press Control-N
Change the input device from Keyboard to Wico Trackball using the space-bar.
Press Return, and select SLOT -2
Press Esc to return to the game

mame apple2p -sl7 wicotrackball apple2_flop_orig:matezone
Press Control-I
Select: WICO TRACKBALL: 7 and WICO FIREBUTTON: 1
Verify that the trackball and first button works
Press Control-I
Select: WICO TRACKBALL: 7 and WICO FIREBUTTON: 2
Verify that the trackball and second button works

To test the Sirius JoyPort with 4 paddles:
mame apple2p -gameio joyport_paddles foosball
When prompted for the player count, press 4 to select 4 player mode
Verify that each of the four paddles can control a distinct set of player rows
